CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.16)
 project(mrover)
 
 ## Compile as C++17, supported in ROS Kinetic and newer, enable some static analysis
-set(MROVER_CPP_COMPILE_OPTIONS -std=c++17 -Wall -Wextra -Werror -pedantic)
+set(MROVER_CPP_COMPILE_OPTIONS -std=c++17)
```

Comment:
We should be able to still use these, ideally anything this complains about should be cleaned up. Were there significant issues that you're worried about for this?

Reply:
It complains about this assert(0 <= maxPWM); on line 216 of Controller.cpp. I'll fix and test all of it tomorrow.